src/snyk/snykCode/views/issueTreeProvider.ts

```diff
@@ -45,143 +40,55 @@ export class IssueTreeProvider extends AnalysisTreeNodeProvder {
   getRootChildren(): TreeNode[] {
     const review: TreeNode[] = [];
     let nIssues = 0;
-    if (!this.contextService.shouldShowCodeAnalysis) return review;
 
-    if (this.snykCode.hasTransientError) {
-      return this.getTransientErrorTreeNodes();
-    } else if (this.snykCode.hasError) {
+    if (!this.contextService.shouldShowCodeAnalysis) return review;
+    if (!this.codeService.isLsDownloadSuccessful) {
```

Comment:
Isn't `isLsDownloadSuccessful` a function? I think you need parenthesis

Reply:
It's defined as a getter function that doesn't require parentheses.